scarlet/source.py

```diff
@@ -333,23 +333,26 @@ def update(self):
             it = self._parent.it
         # Update the central pixel location (pixel_center)
         update.fit_pixel_center(self)
-        if it > self.delay_thresh:
-            update.threshold(self)
+        # Thresholding needs to be fixed (DM-10190)
```

Comment:
So, we're running without thresholding and bboxes right now. How bad is that and what needs to get done to use it again?

Reply:
We never really tested this algorithm for robustness and when running on the HSC patches during the sprint we saw a number of cases, primarily for resolved extended objects where the bulge and disk had very different magnitudes, where the thresholding algorithm would entirely wipe out the disk.

So we need to revisit this algorithm again (I have the opened the referenced JIRA ticket for this) and decide the best way to incorporate some information about the background in the calculation so that we don't cut too high. In the meantime I would recommend going with the attitude that "it's better to do things correctly than fast" and just skip thresholding until we can fix it and test it more rigorously.